packages/block-library/src/common.scss

```diff
@@ -6,6 +6,9 @@
 	@include background-colors-deprecated();
 	@include foreground-colors-deprecated();
 	@include gradient-colors-deprecated();
+	--wp--preset--font-size--normal: 16px;
+	--wp--preset--font-size--huge: 42px;
+	@include font-sizes-deprecated();
```

Comment:
By doing this, the classes will now be `:root .has-font-size-<slug>` and the specificity 020, which increases it from before (010) where we only used the class.

Reply:
At https://github.com/WordPress/gutenberg/pull/37381/commits/669f4f01dcbefaec9ffee717c060b69ff7afc145 removed them from the mixin and added below: it keeps its specificity and work like the other legacy font sizes defined below.